requirements.txt

```diff
@@ -16,7 +16,7 @@ python-Levenshtein~=0.23.0
 python-dateutil~=2.8.2
 wget~=3.2
 stringcase~=1.2.0
-paho-mqtt~=1.6.1
+paho-mqtt>=2.0.0
```

Comment:
Could you please change it to `paho-mqtt~=2.0.0` instead of `paho-mqtt>=2.0.0` otherwise the code might not work if there is a major release.